File: 30daysOFcode_TypeScript/day 02.ts
function solve(meal_cost: number, tip_percent: number, tax_percent: number): void {
    let TotalCost: number = meal_cost + (meal_cost * tip_percent/100) + (meal_cost * tax_percent/100);

    // console.log() to print to stdout
    console.log('The total meal cost is ' + TotalCost + ' dollars.');
}

File: 30daysOFcode_Swift/day 02.swift
import Foundation

let mealCost = Double(readLine()!)!; // original meal price
let tipPercent = Double(readLine()!)!; // tip percentage
let taxPercent = Double(readLine()!)!; // tax percentage

// Calculate Tax and Tip doubles:
let tip = mealCost * tipPercent / 100;
let tax = mealCost * taxPercent / 100;

// cast the result of the rounding operation to an int and save it as totalCost 
let totalCost = Int(round(mealCost + tax + tip));

print("The total meal cost is \(totalCost) dollars.");
